pkg/volume/csi/csi_plugin.go

```diff
@@ -211,11 +210,8 @@ func (p *csiPlugin) Init(host volume.VolumeHost) error {
 		if csiClient == nil {
```

Comment:
Looks like we don't need this code that gets the KubeClient at all anymore after moving the informers.

but also nit: `kubeClient` not `csiClient`

Reply:
I think this is the part that's breaking standalone kubelet in the gke tests

Reply:
Thanks!  Changed this.